index.html

```diff
@@ -17,75 +17,89 @@
   <body>
     <main>
       <header>
+        <div class="nav-title">
+          <h3><a href="index.html">Ismail Changezi</a></h3>
+        </div>
         <nav>
-          <h3>Ismail Changezi</h3>
+          <ul class="nav-items">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li>
+              <a href="#"><img src="img/mail.png" alt="" /></a>
+            </li>
+          </ul>
           <i class="fa-solid fa-bars"></i>
         </nav>
-        <section class="header-section">
+      </header>
+      <section class="header-section">
+        <div class="hero-desc">
           <h4>Hey There.<br />I'm Ismail</h4>
           <h6>I'm a Software Developer</h6>
           <p>
             I can help you build a product , feature or website Look through
             some of my work and experience! If you like what you see and have a
             project you need coded, don’t hestiate to contact me.
           </p>
-          <ul class="icons">
-            <li>
-              <a href="#"> <i class="fa-brands fa-github"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-linkedin-in"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-angellist"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-facebook"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-twitter"></i></a>
-            </li>
-          </ul>
-        </section>
-      </header>
+        </div>
+
+        <ul class="icons">
+          <li>
```

Comment:
- [ ] please ensure that the mobile version `get in touch` button is centralized
![Screenshot (408)_LI](https://user-images.githubusercontent.com/65298048/188022082-0884f21a-340f-4a0a-b31c-b9ba0d1ae536.jpg)

Reply:
Done